README.md

```diff
@@ -1,2 +1,57 @@
 # apple-website-clone
-This project is a clone of the old apple website.
+This project is a clone of the [apple.com](https://web.archive.org/web/20140228164946/http://www.apple.com/) Apple Archived website
+
```

Comment:
- [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it..